db-tests/src/db_tests.rs

```diff
@@ -426,7 +426,7 @@ async fn get_bsos_newer() -> Result<()> {
 async fn get_bsos_sort() -> Result<()> {
     let db = db().await?;
 
-    let uid = 1;
+    let uid = 3;
```

Comment:
? Is this just an mystery value that needs to be unique? (Might want to either create a function that ensures each new value is unique or at least comment to indicate that it should be unique.)

Reply:
This was for debugging, I've reverted it and added https://github.com/mozilla-services/syncstorage-rs/issues/170#issuecomment-521444232

Reply:
Some of the earlier tests left junk in the database that made later tests fail. Assigning unique user ids each time makes them pass.